### PR TITLE
Add initial documentation

### LINKED ISSUE #1: Add documentation
Currently a `README.md` is missing that clarifies how the tool can be

* installed
* executed
* used in general for managing your changelogs of a project

### PR DESCRIPTION
**Pull request by lxanders**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,112 @@
...
+```
+
+To fully profit from `pr-log` your GitHub project needs some small configuration:
+
+* Create the labels mentioned above (you can create GitHub labels from `Issues -> Labels -> New Label`)
+* Set the correct labels on your pull requests
+* Use correct semver versioning for your tags (e.g. `2.4.7`)
+
+To setup the categories of the project correctly that will be used for the changelog, there 
+
```
Comment (lo1tuma) [inline README.md]: You actually HAVE TO set exactly one of those labels per pull request.
There will be an error thrown if you use none of the valid labels or if you use more than one.

**Comment by lo1tuma** [comment]
Can you mention somewhere that the github remote have to be defined in the package.json in following format:

```json
"repository": {
        "type": "git",
        "url": "https://github.com/lo1tuma/pr-log.git"
 }
```

**Comment by lo1tuma** [comment]
Awesome work, thanks. Just added a few comments.

**Comment by lo1tuma** [comment]
Maybe we should also mention that this tool makes only sense if you work completely with pull requests and never commit to master directly.

**Comment by lxanders** [comment]
Thanks for commenting. I added the points you mentioned and made some minor improvements.

**Pull request merged**